service/history/shardContext.go

```diff
@@ -536,6 +557,18 @@ func acquireShard(shardID int, svc service.Service, shardManager persistence.Sha
 	updatedShardInfo := copyShardInfo(shardInfo)
 	updatedShardInfo.Owner = owner
 
+	// initialize the cluster current time to be the same as ack level
+	standbyClusterCurrentTime := make(map[string]time.Time)
+	for clusterName := range svc.GetClusterMetadata().GetAllClusterFailoverVersions() {
+		if clusterName != svc.GetClusterMetadata().GetCurrentClusterName() {
+			if currentTime, ok := shardInfo.ClusterTimerAckLevel[clusterName]; ok {
+				standbyClusterCurrentTime[clusterName] = currentTime
+			} else {
+				standbyClusterCurrentTime[clusterName] = shardInfo.TimerAckLevel
```

Comment:
Why are we using the current cluster ack level to initialize the standby cluster?

Reply:
there will be case that when a new cluster is added, there is no existing ack level for it, so we have to use the current cluster's ack level